### PR TITLE
Fix 'Invalid' state for DatePicker

### PR DESCRIPTION
**Pull request by safronenko23**
## PR Checklist

Check these before submitting your pull request:

- [ ] Visual and behavioral components are separated
- [ ] Exported components are documented with examples
- [ ] Props have JSDoc comments
- [ ] All relevant visual sub-components can be overridden via props
- [ ] Any extra props are spread into the outermost rendered element

## Breaking Changes

Providing 'invalid=true' for DatePicker throws exception. Since get() deprecated and not imported. Changed it to var();

## Changes to Existing Components

Fix 'Invalid' state for DatePicker.

### CODE REVIEW

File: src/components/DatePicker/styles/Wrapper.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import styled, { css, keyframes } from 'styled-components';
+import get from 'extensions/themeGet';
```
Comment (denysYefremov) [inline src/components/DatePicker/styles/Wrapper.js]: As I can see from the code, we mixed the `${get()}` and `var` here, I don't like that idea and think that we should choose one of the approaches. I'd prefer `var`

**Comment by a-type** [comment]
@zelitzsi did we actually deprecate `get`? I thought it was required for IE11, but I don't remember where we came down on that. I'm good with dropping it, especially if we don't want to commit to providing a perfect experience in IE (everything will still be there, but the colors / sizings would be off if CSS vars don't work).

**Comment by zelitzsi** [comment]
@a-type I left in support for `themeGet` primarily for compatibility. It would be nice to have a compatibility mode if it was easy to do, but otherwise I'm okay with using css vars.

**Comment by safronenko23** [comment]
@a-type @zelitzsi let me know if I need to change something.

**Pull request merged**